ansible/ci/check_eessi.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Run EESSI test job
+  hosts: login # TODO: Limit this to single node when there are multiple?
```

Comment:
I think you can just use `login[0]` to do this, see https://docs.ansible.com/ansible/latest/inventory_guide/intro_patterns.html#using-group-position-in-patterns. Definitely worth adding to avoid running multiple times.